### DISCUSSION TITLE
parameters

### DISCUSSION THREAD

**Discussion by cgencer** [spatie/ssh#49] [Ideas]
Hey, nice package, thanks!
Just a very small idea for next addition: execute command could accept an array as parameters, to be passed to the command-line, thus removing the need to stringify all parameters...
thnaks again for your contributiion to the community... spatie components are always favoured here! :P

**Comment by freekmurze** [comment]
Feel free to send a PR with tests and updated readme for this. I'll merge it in if the code to achieve this is simple 👍
